pkg/kubelet/kubelet_test.go

```diff
@@ -1993,6 +2009,14 @@ func TestHandlePodResourcesResize(t *testing.T) {
 	defer kubelet.podManager.DeletePod(testPod2)
 	defer kubelet.podManager.DeletePod(testPod1)
 
+	kubeclientmanager.NewKubeClientManager()
+	kubeclientmanager.ClientManager.RegisterTenantSourceServer(
+		"api",
+		&v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Tenant: "system",
+			}})
```

Comment:
same comment as above.

Besides, do we need non-system tenant test cases?

Reply:
kubelet_test does not have any tenant test yet and it should need them. I'll create an issue to track this.